WillTrem/UInnovate
# 238 us i18n create i18n call in the vmd

- I adde the logic for the language change
- I added a few label in the navbar and the settings for french and english

to test:

- make sure to refresh db with use case 1. run from the script folder  "./db.sh -r -f '/useCase1'  " 
- Go to settings
- should be able to toggle between languages

![I18n_language_selection_lbl](https://github.com/WillTrem/UInnovate/assets/10947924/8bd67f27-e933-4b58-88fd-e902298bf14d)
